src/Php54/bootstrap.php

```diff
@@ -16,7 +16,14 @@
         function trait_exists($class, $autoload = true) { return $autoload && class_exists($class, $autoload) && false; }
     }
     if (!function_exists('class_uses')) {
-        function class_uses($class, $autoload = true) { return $autoload && class_exists($class, $autoload) && false; }
+        function class_uses($class, $autoload = true)
+        {
+            if (is_object($class) || class_exists($class, $autoload) || interface_exists($class, $autoload)) {
```

Comment:
interface_exists($class, **false**) because the autoloader has already been triggered, if needed, by class_exists